scripts/update_and_rebuild_libmesh.sh

```diff
@@ -28,6 +28,21 @@ if [[ $? == 0 && "x$git_dir" == "x" ]]; then
   fi
 fi
 
+# check if the user has ccache configured
+DISABLE_TIMESTAMPS=""
+echo $CXX | cut -d ' ' -f1 | grep '^ccache$' > /dev/null
+if [ $? == 0 ]; then
+  echo -n "ccache detected - "
+  # check if timestamps are explicitly enabled
+  echo "$* " | grep -- '--enable-timestamps ' > /dev/null
+  if [ $? == 0 ]; then
+    echo "warning: setting --enable-timestamps explicitly will negatively impact the ccache performance"
+  else
+    echo "configuring limbesh with --disable-timestamps to improve cache hit rate"
+    DISABLE_TIMESTAMPS="--disable-timestamps"
+  fi
```

Comment:
This is nice of you to respect the user's options, but maybe we should just append `--disable-timestamps` to `$*` no matter what when ccache is detected... ccache is basically useless without that, right?